### ISSUE TITLE
[BUG] Quiver fix and improvement

### ISSUE THREAD

**Issue by u3o** [pyroclient/pyro#234]
**Describe the bug**
Sometimes quivet pull on aunt for so little and arrow doesnt appear. Or arrows sometimes hit you and then drop as an item without giving an effect to you (maybe thats a minecraft bug)

**To Reproduce**
1. Turn on quiver a couple of times
2. See error

**Expected behavior**
Quivet shoots to you until at least 1 arrow of each effect hits you

**Screenshots**
Add screenshots or videos to help explain the problem.

**Additional context**
Also it would be great to add option to change how long does the bow pull on aunt (?)

**Comment by Artamedes** [comment]
If you use it too much in quick succession you will flag NCP for too many inventory clicks

**Comment by u3o** [comment]
> If you use it too much in quick succession you will flag NCP for too many inventory clicks

No, it happens noy after a lot of times turning on the module, it is just random. You can not use it for 5 minutes and have this issues
